src/bin/oura/watch.rs

```diff
@@ -152,10 +158,17 @@ pub(crate) fn command_definition<'a>() -> clap::Command<'a> {
                 .takes_value(true)
                 .help("milliseconds to wait between output lines (for easier reading)"),
         )
+        .arg(
+            clap::Arg::new("terminal_width")
```

Comment:
Isn't the convention for flags to be kebab case? So `terminal-width`

Reply:
updated